Coders/DSA Concepts/Armstrong number.java

```diff
@@ -0,0 +1,21 @@
+public class Armstrong {
+
+    public static void main(String[] args) {
+
+        int number = 371, originalNumber, remainder, result = 0;
+
+        originalNumber = number;
+
+        while (originalNumber != 0)
+        {
+            remainder = originalNumber % 10;
+            result += Math.pow(remainder, 3);
```

Comment:
[Armstrong_number.txt](https://github.com/SR-Sunny-Raj/Hacktoberfest2022-for-everyone/files/9843745/Armstrong_number.txt)